### ISSUE TITLE
Webapi.FormData.t should not be a record type

### ISSUE THREAD

**Issue by yawaramin** [tinymce/rescript-webapi#84]
It is currently defined as:

```
type t = private {
  keys: Webapi__Iterator.t<string>,
  values: Webapi__Iterator.t<EntryValue.t>,
  entries: Webapi__Iterator.t<(string, EntryValue.t)>,
}
...
@send external keys: t => Webapi__Iterator.t<string> = "keys"
@send external values: t => Webapi__Iterator.t<EntryValue.t> = "values"
@send external entries: t => Webapi__Iterator.t<(string, EntryValue.t)> = "entries"
```

Notice that `keys()`, `values()`, and `entries()` are _nullary methods,_ not _getters._ Exposing record fields like `keys: Webapi__Iterator.t<string>` is incorrect because those are not the correct types. This type should be reverted back to an abstract type.

Relates to #2

**Comment by TheSpyder** [comment]
This happened in 2ba24b0a when I added the fetch bindings. They were `@send` before, not `@get`, I don’t know what I was thinking.

**Comment by TheSpyder** [comment]
In theory the record fields could be defined as functions, but in my experience calling a record function _always_ generates a Curry call at runtime. So I agree the type should revert to abstract.

**Comment by TheSpyder** [comment]
Released as 0.5.0
